xbmc/windowing/gbm/GBMUtils.cpp

```diff
@@ -21,18 +21,48 @@
 #include "GBMUtils.h"
 #include "utils/log.h"
 
-bool CGBMUtils::InitGbm(struct gbm *gbm, int hdisplay, int vdisplay)
+bool CGBMUtils::CreateDevice(struct gbm *gbm, int fd)
 {
-  gbm->width = hdisplay;
-  gbm->height = vdisplay;
+  if (gbm->device)
+    CLog::Log(LOGWARNING, "CGBMUtils::%s - device already created", __FUNCTION__);
 
-  gbm->surface = gbm_surface_create(gbm->dev,
+  gbm->device = gbm_create_device(fd);
+  if (!gbm->device)
+  {
+    CLog::Log(LOGERROR, "CGBMUtils::%s - failed to create device", __FUNCTION__);
+    return false;
+  }
+
+  return true;
+}
+
+void CGBMUtils::DestroyDevice(struct gbm *gbm)
+{
+  if (!gbm->device)
+    CLog::Log(LOGWARNING, "CGBMUtils::%s - device already destroyed", __FUNCTION__);
```

Comment:
I don't think that you need to log this case.

Reply:
I agree, will remove

Reply:
Please keep logging, helps spot issues until gbm windowing is fully stable